### PR TITLE
Add more card emoji tests to Guava list

### PR DESCRIPTION
**Pull request by rzrobin213**
Signed-off-by: Robin Zheng <robin.zheng@bnymellon.com>

### CODE REVIEW

File: deck-of-cards-kata-solutions/src/test/java/bnymellon/codekatas/deckofcards/list/immutable/GoogleGuavaDeckOfCardsAsListTest.java [hunk 1]
```diff
@@ -16,13 +16,14 @@
...
+import java.util.Collections;
+import java.util.List;
 import java.util.Random;
 
 import bnymellon.codekatas.deckofcards.Rank;
 import bnymellon.codekatas.deckofcards.Suit;
 import com.google.common.base.Joiner;
-import com.google.common.collect.HashMultiset;
-import com.google.common.collect.Multiset;
+import com.google.common.collect.*;
```
Comment (donraab) [inline deck-of-cards-kata-solutions/src/test/java/bnymellon/codekatas/deckofcards/list/immutable/GoogleGuavaDeckOfCardsAsListTest.java:26]: Please specify individual classes in imports.

File: deck-of-cards-kata-solutions/src/test/java/bnymellon/codekatas/deckofcards/list/immutable/GoogleGuavaDeckOfCardsAsListTest.java [hunk 2]
```diff
@@ -101,6 +105,16 @@ public void shuffleAndDealHands()
         var jdkHands = this.jdkDeck.shuffleAndDeal(new Random(1), 5, 5);
         var ggHands = this.ggDeck.shuffleAndDeal(new Random(1), 5, 5);
         Assertions.assertEquals(jdkHands, ggHands);
+        var hands = Lists.newArrayList(
+                Iterables.concat(Collections.singleton(Collections2.transform(
+                        ggHands, o -> Joiner.on(", ").join(Ordering.natural().sortedCopy(o))))));
+        List<String> expectedHands = org.eclipse.collections.impl.factory.Lists.mutable.with(
```
Comment (donraab) [inline deck-of-cards-kata-solutions/src/test/java/bnymellon/codekatas/deckofcards/list/immutable/GoogleGuavaDeckOfCardsAsListTest.java:111]: Use `List.of` from JDK or `Lists.newArrayList` from Guava for Guava tests.

File: deck-of-cards-kata-solutions/src/test/java/bnymellon/codekatas/deckofcards/list/immutable/GoogleGuavaDeckOfCardsAsListTest.java [hunk 3]
```diff
@@ -111,6 +125,16 @@ public void dealHands()
         var jdkHands = this.jdkDeck.dealHands(jdkShuffled, 5, 5);
         var ggHands = this.ggDeck.dealHands(ggShuffled, 5, 5);
         Assertions.assertEquals(jdkHands, ggHands);
+        var hands = Lists.newArrayList(
+                Iterables.concat(Collections.singleton(Collections2.transform(
+                        ggHands, o -> Joiner.on(", ").join(Ordering.natural().sortedCopy(o))))));
+        List<String> expectedHands = org.eclipse.collections.impl.factory.Lists.mutable.with(
```
Comment (donraab) [inline deck-of-cards-kata-solutions/src/test/java/bnymellon/codekatas/deckofcards/list/immutable/GoogleGuavaDeckOfCardsAsListTest.java:131]: Use `List.of` from JDK or `Lists.newArrayList` from Guava.

File: deck-of-cards-kata/src/test/java/bnymellon/codekatas/deckofcards/list/immutable/GoogleGuavaDeckOfCardsAsListTest.java [hunk 4]
```diff
@@ -101,6 +105,16 @@ public void shuffleAndDealHands()
         var jdkHands = this.jdkDeck.shuffleAndDeal(new Random(1), 5, 5);
         var ggHands = this.ggDeck.shuffleAndDeal(new Random(1), 5, 5);
         Assertions.assertEquals(jdkHands, ggHands);
+        var hands = Lists.newArrayList(
+                Iterables.concat(Collections.singleton(Collections2.transform(
+                        ggHands, o -> Joiner.on(", ").join(Ordering.natural().sortedCopy(o))))));
+        List<String> expectedHands = org.eclipse.collections.impl.factory.Lists.mutable.with(
```
Comment (donraab) [inline deck-of-cards-kata/src/test/java/bnymellon/codekatas/deckofcards/list/immutable/GoogleGuavaDeckOfCardsAsListTest.java:111]: Same comment as above.

File: deck-of-cards-kata/src/test/java/bnymellon/codekatas/deckofcards/list/immutable/GoogleGuavaDeckOfCardsAsListTest.java [hunk 5]
```diff
@@ -111,6 +125,16 @@ public void dealHands()
         var jdkHands = this.jdkDeck.dealHands(jdkShuffled, 5, 5);
         var ggHands = this.ggDeck.dealHands(ggShuffled, 5, 5);
         Assertions.assertEquals(jdkHands, ggHands);
+        var hands = Lists.newArrayList(
+                Iterables.concat(Collections.singleton(Collections2.transform(
+                        ggHands, o -> Joiner.on(", ").join(Ordering.natural().sortedCopy(o))))));
+        List<String> expectedHands = org.eclipse.collections.impl.factory.Lists.mutable.with(
```
Comment (donraab) [inline deck-of-cards-kata/src/test/java/bnymellon/codekatas/deckofcards/list/immutable/GoogleGuavaDeckOfCardsAsListTest.java:131]: Same comment as above.

**Review by donraab — CHANGES_REQUESTED** [review]
Thanks for the the contribution @rzrobin213 ! I've requested a few changes inline.

File: deck-of-cards-kata-solutions/src/test/java/bnymellon/codekatas/deckofcards/list/immutable/GoogleGuavaDeckOfCardsAsListTest.java (hunk 1 above)
Comment (rzrobin213) [reply deck-of-cards-kata-solutions/src/test/java/bnymellon/codekatas/deckofcards/list/immutable/GoogleGuavaDeckOfCardsAsListTest.java]: Done!

File: deck-of-cards-kata-solutions/src/test/java/bnymellon/codekatas/deckofcards/list/immutable/GoogleGuavaDeckOfCardsAsListTest.java (hunk 3 above)
Comment (rzrobin213) [reply deck-of-cards-kata-solutions/src/test/java/bnymellon/codekatas/deckofcards/list/immutable/GoogleGuavaDeckOfCardsAsListTest.java:136]: Removed use of Java.util.Collections library

**Review by donraab — APPROVED** [review]
LGTM! Thanks for the contribution @rzrobin213 !

**Pull request merged**